include/gdsync/core.h

```diff
@@ -350,6 +372,35 @@ int gds_stream_post_descriptors(CUstream stream, size_t n_descs, gds_descriptor_
  */
 int gds_post_descriptors(size_t n_descs, gds_descriptor_t *descs, int flags);
 
+/**
+ * \brief: TODO
+ *
+ *
+ * \param flags - TODO
+ *
+ * \return
+ * 0 on success or one standard errno error
+ *
+ *
+ * Notes:
+ * - TODO.
+ */
+int gds_report_post(struct gds_qp *gqp  /*, struct gds_send_wr* wr*/);
```

Comment:
The function documentation is still missing. I guess this function advances the tracking in the gds_qp struct of the current producer index with the given send wr size?